components/header/Header.tsx

```diff
@@ -9,27 +9,33 @@ import {
   MenuItem,
   Button,
 } from '@mui/material'
-import Image from 'next/image'
+import Image, { StaticImageData } from 'next/image'
 import React from 'react'
 import logoImg from '../../public/images/logo.png'
 import MenuIcon from '@mui/icons-material/Menu'
+import { NextRouter, useRouter } from 'next/router'
 import { pages, siteName } from '../../data/data'
```

Comment:
change the paths into absolute by adding configuration in next.config file base path.